newIDE/app/src/EventsSheet/InlinePopover.js

```diff
@@ -83,7 +84,14 @@ export default function InlinePopover(props: Props) {
           // Note that the content of the popover can capture the event
           // and stop its propagation (for example, the GenericExpressionField
           // when showing autocompletion), which is fine.
+
+          // When the event match with Escape key changes are canceled.
           if (shouldCloseOrCancel(event)) {
+            props.onRequestClose(event);
```

Comment:
This is not very clear: `onRequestClose` will do something different according to if event is passed or not... weird no? 
onRequestClose should do one thing: closing the popover. You can pass it an additional parameter, but in this case prefer a boolean or something with a proper name rather than passing or not event.